include/drivers/uart.h

```diff
@@ -512,6 +512,11 @@ static inline int z_impl_uart_tx_abort(const struct device *dev)
  * immediately. After that event handler, set using @ref uart_callback_set,
  * is called with UART_RX_RDY or UART_RX_BUF_REQUEST events.
  *
+ * UART_RX_RDY event is generated in two cases:
```

Comment:
It is actually described here:
https://github.com/zephyrproject-rtos/zephyr/blob/ab3ce1af453e3f2518cfbba334081636b8f18105/include/drivers/uart.h#L54-L60
Maybe the above section could be just pointed from here instead?

Reply:
:) Just pointed it out: https://github.com/zephyrproject-rtos/zephyr/pull/30859#issuecomment-748006948 
How to point it? Somehow name this section?